docs/deploy/configuration/replica-movement.mdx

```diff
@@ -0,0 +1,324 @@
+
+<Tabs groupId="languages">
+<TabItem value="py" label="Python Client v4">
+<FilteredTextBlock
+text={PyCode}
+startMarker="# START ReplicateShard"
+endMarker="# END ReplicateShard"
+language="py"
+/>
+</TabItem>
+<TabItem value="js" label="JS/TS Client v3">
+
+```typescript
+// JS/TS support coming soon
+```
+
+</TabItem>
+<TabItem value="go" label="Go">
+
+```go
+// Go support coming soon
+```
+
+</TabItem>
+<TabItem value="java" label="Java">
+
+```java
+// Java support coming soon
+```
+
+</TabItem>
+</Tabs>
+
+<details>
+  <summary>Code output</summary>
+
+```
+Replication initiated, ID: 32536c0e-09e1-4ea1-a2c5-e85af10a9d58
+```
+
+</details>
+
+## Checking the status of a replication operation
+
+Since movement is asynchronous, you can monitor its progress. The movement operation can have one of the following states:
+
+- `REGISTERED`
+- `HYDRATING`
+- `FINALIZING`
+- `DEHYDRATING`
+- `READY`
+- `CANCELLED`
+
+To learn more about the replication states, check out [Concepts: Cluster architecture](/docs/weaviate/concepts/replication-architecture/cluster-architecture.md#replica-movement).
+
+<Tabs groupId="languages">
+<TabItem value="py" label="Python Client v4">
+<FilteredTextBlock
+text={PyCode}
+startMarker="# START CheckOperationStatus"
+endMarker="# END CheckOperationStatus"
+language="py"
+/>
+</TabItem>
+<TabItem value="js" label="JS/TS Client v3">
+
+```typescript
+// JS/TS support coming soon
+```
+
+</TabItem>
+<TabItem value="go" label="Go">
+
+```go
+// Go support coming soon
+```
+
+</TabItem>
+<TabItem value="java" label="Java">
+
+```java
+// Java support coming soon
+```
+
+</TabItem>
+</Tabs>
+
+<details>
+  <summary>Code output</summary>
+
+```
+Status for 32536c0e-09e1-4ea1-a2c5-e85af10a9d58: REGISTERED
+```
+
+</details>
+
+## Listing replication operations
+
+List all ongoing or recently completed operations. This can be filtered by node, collection and shard.
+
+<Tabs groupId="languages">
+<TabItem value="py" label="Python Client v4">
+<FilteredTextBlock
+text={PyCode}
+startMarker="# START ListReplicationOperations"
+endMarker="# END ListReplicationOperations"
+language="py"
+/>
+</TabItem>
+<TabItem value="js" label="JS/TS Client v3">
+
+```typescript
+// JS/TS support coming soon
+```
+
+</TabItem>
+<TabItem value="go" label="Go">
+
+```go
+// Go support coming soon
+```
+
+</TabItem>
+<TabItem value="java" label="Java">
+
+```java
+// Java support coming soon
+```
+
+</TabItem>
+</Tabs>
+
+<details>
+  <summary>Code output</summary>
+
+```
+Total replication operations: 1
+Filtered operations for collection 'MyReplicatedDocCollection' on 'node3': 1
+```
+
+</details>
+
+## Cancelling a replication operation
+
+If a movement needs to be stopped before completion. The operation will be stopped and will remain in a `CANCELLED` state.
```

Comment:
The operation will be stopped if possible. If successfully cancelled, will be logged as `CANCELLED`.